python/get_unlinked_top_containers.py

```diff
@@ -0,0 +1,39 @@
+import asnake.logging as logging
+from asnake.client import ASnakeClient
+
+logging.setup_logging(filename="archivessnake.log", level="INFO")
+# set label for custom logger - all output will be in archivessnake.log
+logger = logging.get_logger("get_unlinked_top_containers")
+client = ASnakeClient()
+
+
+def get_unlinked_top_containers():
+    all_top_containers = client.get(
+        "repositories/2/top_containers", params={"all_ids": True}
+    ).json()
+    total_top_containers = len(all_top_containers)
+    logger.info(f"Total top containers: {total_top_containers}")
+    pages = (total_top_containers // 250) + 1
```

Comment:
The "magic number" of 250 gets used twice (here and line 23).  The [API docs](https://archivesspace.github.io/archivesspace/api/#paginated-endpoints) say "default set in AppConfig" - is that exposed via something we can set in `archivessnake.yml`?  

I also see the [low-level API](https://github.com/archivesspace-labs/ArchivesSnake?tab=readme-ov-file#low-level-api) mentions `get_paged` which seems to handle paginated data for us.

If neither of those is useful for us, then add an optional `pagesize` parameter to `get_unlinked_top_container()`, and set the default value there to 250.  That would also require adding `argparse` or similar for command-line argument support.